src/main/java/com/auth0/client/mgmt/ManagementAPI.java

```diff
@@ -45,6 +46,18 @@ public ManagementAPI(String domain, String apiToken) {
                 .build();
     }
 
+    /**
+     * Update the API token to use on new calls. This is useful when the token is about to expire or it already has.
+     * Please note you'll need to obtain the correspondent entity again for this to apply. e.g. call {@link #clients()} again.
+     * See the Management API section in the readme or visit https://auth0.com/docs/api/management/v2/tokens to learn how to obtain a token.
+     *
+     * @param apiToken the token to authenticate the calls with.
+     */
+    public void setApiToken(String apiToken) {
+        Asserts.assertNotNull(apiToken, "api token");
```

Comment:
I might be wrong but it looks like you've got these params backwards?

http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertNotNull(java.lang.Object)

I'm not sure if `Asserts.assertNotNull()` is the same one.

Reply:
I think I'm wrong ... `Assert` is a unit testing thing, `Asserts` is in this lib

Reply:
Yes! `Asserts` is a helper class that throws an exception if a value is different than the expected. I think I mentioned this on one of my reviews to your PRs last week. It's basically to avoid boilerplate code.

```java
if (value == null) {
   throw new IllegalArgumentException(String.format("'%s' cannot be null!", name));
}
```